qutebrowser/browser/hints.py

```diff
@@ -581,9 +580,8 @@ def _spawn(self, url, context):
         """
         urlstr = url.toString(QUrl.FullyEncoded | QUrl.RemovePassword)
         args = context.get_args(urlstr)
-        cmd, *args = args
-        proc = guiprocess.GUIProcess(self._win_id, what='command', parent=self)
-        proc.start(cmd, args)
+        commandrunner = runners.CommandRunner(self._win_id)
+        commandrunner.run_safely('spawn ' + ' '.join(args))
```

Comment:
Clever solution! I think it'd need to call [`shlex.quote`](https://docs.python.org/dev/library/shlex.html#shlex.quote) on the arguments though, so it works fine with a commandline containing spaces? I'm not 100% sure though with all the different splitting involved, would be nice to have a test added.